examples/llama2/requirements.txt

```diff
@@ -1,6 +1,7 @@
 datasets>=2.8.0
-git+https://github.com/huggingface/optimum.git
+optimum>=1.17.0
 onnx>=1.14.0
 protobuf==3.20.2
 torch -i https://download.pytorch.org/whl/nightly/cu118
 transformers>=4.33.2
+auto-gptq
```

Comment:
can you put this at the top of the list so that it is alphabetically sorted. Weird since the linter should have caught it, @guotuofeng?

Reply:
Linter will not catch it as of now. It was caught by pre-commit I guess. 
![image](https://github.com/microsoft/Olive/assets/13343117/3b93dd6b-62eb-4c9e-9ed6-477903887752)

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/4fa224e0-902b-4a73-8b4a-cc5833df6340)
Oh, sry, we also have that in linter runner. `lintrunner -a` can fix but somehow the checking passed even if the packages are not in order.

Reply:
> can you put this at the top of the list so that it is alphabetically sorted. Weird since the linter should have caught it, @guotuofeng?

I will let @Taka152 investigate why the lintrunner does't raise issue. If it is bug, we will contact the owner to fix.

Reply:
this seems lintrunner-adapter's bug.